.github/workflows/release.yml

```diff
@@ -49,6 +49,24 @@ jobs:
         id: changesets
         with:
           publish: yarn changeset publish
+          version: yarn changeset:version:custom
```

Comment:
Overrides the standard `changeset version` command with our own. You can test this out by running it in the root of the mono repo.